guides/common/modules/proc_adding-custom-deb-repositories.adoc

```diff
@@ -50,7 +50,8 @@ For more information, see xref:Download_Policies_Overview_{context}[].
 . From the *Mirroring Policy* list, select the type of content synchronization {ProjectServer} performs.
 For more information, see xref:Mirroring_Policies_Overview_{context}[].
 . Optional: In the *HTTP Proxy Policy* field, select an HTTP proxy.
-. Optional: If you want, you can clear the *Publish via HTTP* check box to disable this repository from publishing through HTTP.
+. Optional: You can clear the *Unprotected* check box to require a subscription entitlement certificate for accessing this repository.
+By default, the repository is published through HTTP.
```

Comment:
Do we allow both HTTP and HTTPS? I noticed in another place that only HTTPS was allowed for security reasons and I wonder what the case is here.

Reply:
I have tested this on Foreman 3.2 and it's also available through HTTP & I found no setting to prohibit downloads over HTTP. Do you know more @m-bucher ?

Reply:
I do not know if there is a special handling for RedHat repos, but the repository will always be distributed via HTTPS where clients can authenticate using subscription-manager.
But if you want/have to access the repository without authentication (e.g. because the OS needs a special repository to install subscription-manager in the first place), it has to be offered via HTTP, because HTTPS-repos always require client-atuhentication using SSL-certs.
For Debian repositories distributing over HTTP is perfectly save as far as authenticity is concerned (as long as a secure GPG-key has been used for signature).

Reply:
So maybe we should mention both and recommend HTTPS as preferred? Or mention cases when HTTP is necessary?

Reply:
I have tested this again. It's possible to access content through HTTP.